content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -272,16 +275,66 @@ transaction functions. We recommend you get in and out as quick as possible!
 
 - [Default transaction isolation level in PostgreSQL](https://www.postgresql.org/docs/9.3/runtime-config-client.html#GUC-DEFAULT-TRANSACTION-ISOLATION)
 - [Default transaction isolation level in Microsoft SQL Server](https://docs.microsoft.com/en-us/sql/t-sql/statements/set-transaction-isolation-level-transact-sql?view=sql-server-ver15)
 - [Default transaction isolation level in MySQL](https://dev.mysql.com/doc/refman/8.0/en/innodb-transaction-isolation-levels.html)
 
-## Join the Conversation on GitHub
+</Admonition>
+
+To set the transaction isolation level, use the `isolationLevel` option in the second parameter of the API:
+
+For example:
+
+```jsx
+await prisma.$transaction(
+  async (prisma) => {
+    // Code running in a transaction...
+  },
+  {
+    isolationLevel: Prisma.TransactionIsolationLevel.Serializable, // optional
+    maxWait: 5000, // default: 2000
+    timeout: 10000, // default: 5000
+  }
+)
+```
+
+Prisma Client supports the following isolation levels if they are available in the underlying database:
+
+- `ReadCommitted`
+- `ReadUncommitted`
+- `RepeatableRead`
+- `Serializable`
+- `Snapshot`
 
-If you'd like to see transactions supported in the future, [please join the discussion on GitHub](https://github.com/prisma/prisma-client-js/issues/349).
+##### Isolation levels by database
+
+The isolation levels available for each database connector are as follows:
+
+| Database    | `ReadCommitted` | `ReadUncommitted` | `RepeatableRead` | `Serializable` | `Snapshot` |
+| ----------- | --------------- | ----------------- | ---------------- | -------------- | ---------- |
+| PostgreSQL  | ✔️              | ✔️                | ✔️               | ✔️             | No         |
+| MySQL       | ✔️              | ✔️                | ✔️               | ✔️             | No         |
+| SQL Server  | ✔️              | ✔️                | ✔️               | ✔️             | ✔️         |
+| CockroachDB | No              | No                | No               | ✔️             | No         |
+| SQLite      | No              | No                | No               | ✔️             | No         |
+
+By default Prisma Client will set the isolation level to the value currently configured in your database. For database-specific information on isolation levels, including database defaults, see the following resources:<br /><br />
+
+- [Default transaction isolation level in PostgreSQL](https://www.postgresql.org/docs/9.3/runtime-config-client.html#GUC-DEFAULT-TRANSACTION-ISOLATION)
+- [Default transaction isolation level in Microsoft SQL Server](https://docs.microsoft.com/en-us/sql/t-sql/statements/set-transaction-isolation-level-transact-sql?view=sql-server-ver15)
+- [Default transaction isolation level in MySQL](https://dev.mysql.com/doc/refman/8.0/en/innodb-transaction-isolation-levels.html)
```

Comment:
What about CockroachDB and SQLite?

Reply:
They both support only single isolation level.

Reply:
Added a note to mention this.